api/swagger/src/paths/search/string.yaml

```diff
@@ -0,0 +1,35 @@
+x-swagger-router-controller: search
+get:
+  description: Returns entities matching the search criteria
+  operationId: searchString
+  parameters:
+    - name: q
+      in: query
+      description: Query string
+      required: true
+      type: string
+    - name: from
+      in: query
+      description: Return results after a certain point
+      required: false
+      type: integer
+      format: int32
+    - name: size
+      in: query
+      description: Only return (size) entries
+      required: false
+      type: integer
+      format: int32
+  responses:
+    "200":
+      description: Success
+      schema:
+        $ref: '#/definitions/SearchResults'
+    "404":
+      description: No Results
```

Comment:
Ehhhhhhhhhhhhhh, I'd strongly urge @syncall to overthink his specification…

404 usually indicates, that the resource wasn't found, not that the query didn't have any results...